src/qibosoq/__main__.py

```diff
@@ -27,9 +27,26 @@ def define_logger():
     return new_logger
 
 
-logger = define_logger()
+def define_program_logger():
+    """Define qick logger format and handler"""
 
-HOST = "192.168.0.72"  # Server address
+    new_logger = logging.getLogger("qick_program")
+    new_logger.setLevel(logging.DEBUG)
+
+    filename = "/home/xilinx/logs/program.log"
+    formatter = logging.Formatter("%(levelname)s :: %(asctime)s\n%(message)s", "%Y-%m-%d %H:%M:%S")
+
+    handler = logging.handlers.RotatingFileHandler(filename, mode="w", backupCount=3)
+    handler.setFormatter(formatter)
+
+    new_logger.addHandler(handler)
+    return new_logger
+
+
+logger = define_main_logger()
+program_logger = define_program_logger()
+
+HOST = "192.168.0.81"  # Server address
```

Comment:
As said on the client side: this could be an [`ipadress`](https://docs.python.org/3/library/ipaddress.html)

Reply:
Sorry I didn't comment on this suggestion, seems a bit of an overkill to me... 
In the end if someone tries to set something that is not valid it will just fail